frontend/src/components/LoginStatus.vue

```diff
@@ -6,10 +6,10 @@
     <div
       v-if="username"
     >
-      User: {{ username }}
+      {{ $t('login.user', { username: username }) }}
```

Comment:
Wow I didn't know it can work this way, I'd have written it:
`{{ $t('login.user') }} {{ username }}` :D 💎